templates/base/_package.json

```diff
@@ -29,14 +34,8 @@
       "pre-commit": "{{huskyConfig}}"
     }
   },
-  "dependencies": {
-  },
-  "peerDependencies": {
```

Comment:
There is no need for `peerDependencies` since consumer will have `react` and other required packages as `dependencies`. The build process will exclude them from final bundle. However, they will still be available as transient dependencies in final consuming application.